presto-native-execution/presto_cpp/main/types/PrestoToVeloxExpr.cpp

```diff
@@ -411,7 +415,7 @@ std::shared_ptr<const ConstantTypedExpr> VeloxExprConverter::toVeloxExpr(
     default: {
       const auto value = getConstantValue(type, pexpr->valueBlock);
 
-      return std::make_shared<ConstantTypedExpr>(value);
+      return std::make_shared<ConstantTypedExpr>(type, value);
```

Comment:
Hi @aditi-pandit. I made the change based on your comment. Is this what you originally meant? 
Thanks.

Reply:
Yes. Thanks !